### DISCUSSION TITLE
[Suggestion] add compatibility table of supported RNR version of each minor version React Native

### DISCUSSION THREAD

**Discussion by flixyudh** [software-mansion/react-native-reanimated#4991] [General]
I just init React Native 0.71.13, when i try to build android with newArch enabled, i got an error

> [react-native-reanimated] Reanimated 3.4.2 supports the New Architecture only on the latest minor release of React Native. Please upgrade to React Native 0.72.0+ or downgrade to an older version of Reanimated v3.

i try to find a backward compatibility in [README.md](https://github.com/software-mansion/react-native-reanimated/blob/main/README.md) but not found

i think it's good to add a compatibility version supported of RNR for each React Native something like [Code Push did](https://github.com/microsoft/react-native-code-push/#supported-react-native-platforms)

is it acceptable?

![github com_microsoft_react-native-code-push_](https://github.com/software-mansion/react-native-reanimated/assets/19633881/f10dbfed-6f89-437f-a313-21843be41643)

**Comment by tjzel** [comment]
Hi @zxccvvv, thanks for posting it. I've been working on new Troubleshooting section in our docs lately and I noticed we didn't have it - we plan to do this in the near future - now that you asked we have double the motivation.
